api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3056,6 +3056,47 @@ message DatabaseSessionQuery {
   ];
 }
 
+// DatabasePermissionUpdate is emitted when a user database permissions are updated.
+message DatabasePermissionUpdate {
```

Comment:
Double checking: should new audit events adhere to [new resource guidelines](https://github.com/gravitational/teleport/blob/master/rfd/0153-resource-guidelines.md?plain=1#L1), or should they stick with the de facto legacy standards? cc @rosstimothy

Reply:
Events are not resources, so I don't think RFD 153 applies here. I wouldn't mind rewriting resources as plain protos, without the need for gogoproto, but I think this should be a distinct effort.